pyccel/parser/syntactic.py

```diff
@@ -499,13 +499,13 @@ def _visit_BinOp(self, stmt):
         second = self._visit(stmt.right)
 
         if isinstance(stmt.op, ast.Add):
-            return PyccelAdd(first, second)
+            return PyccelAdd(first, second, simplify = True)
```

Comment:
Only things in the codegen stage, and maybe the semantic stage should be simplified. The aim is to preserve the user's code where possible. So if the user writes:
```
a = x+(-5)
```
then maybe he has a reason for this strange syntax, so we want to print:
```
a = x+(-5)
```

but if the user writes:
```
a = randint(-x, y)
```
there isn't an equivalent for this function in fortran. Currently this means that we print:
```
a = floor((y - (-x)) * Dummy_0001 + (-x), kind=C_INT64_T)
```
but in this case the strange syntax is our fault. So it should be simplified to:
```
a = floor((y + x) * Dummy_0001 - x, kind=C_INT64_T)
```